output_template/python/usb_camera_demo.py

```diff
@@ -79,6 +80,32 @@ def add_class_label(canvas,
     cv2.putText(canvas, text, dl_corner, font, font_scale, font_color, line_type)
 
 
+def label_to_color_image(results, colormap):
+    """Adds color defined by the dataset colormap to the label.
```

Comment:
the colormap is not defined buy the dataset.

Reply:
fixed comment with commit https://github.com/blue-oil/blueoil/pull/155/commits/7c71b86adeb9caafc3552b7ebe61a4ab912f3c61